Lab5.py

```diff
@@ -8,12 +8,20 @@ def Greeting (self):
         print("Hello, my name is " + self.name)
 
     #def Save (self):
-    #    pass
+    #    print("Saved")
```

Comment:
this is not the intention of this function to print

Reply:
The task is to show SOLID principles and I show it with print element to show what method is running. You want me to make real save algoritm? If yes, why? The save algorithm is not part of SOLID principles.

Reply:
Am I right?

Reply:
look, you've done enough to get a mark, but in case you want to learn anything you have to do extra

Reply:
and btw, in case you don't know, implement this function shuold've taken less time that answering me in this pull request